src/App.tsx

```
@@ -2,27 +2,65 @@
 	const token = localStorage.getItem('authToken');
 
 	useEffect(() => {
 		if (token) {
-			setRoutes(authRoutes);
-			const redirectUrl = localStorage.getItem('redirectUrl');
-			if (redirectUrl) {
-				window.location.href = redirectUrl;
-				localStorage.removeItem('redirectUrl');
+			const decoded = jwtDecode(token) as DecodedToken;
+			// Check for roles in resource_access
+			const resourceAccess = decoded.resource_access || {};
```

ESLint

[error] 'resourceAccess' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)